app/controllers/chatfuel_controller.rb

```diff
@@ -53,6 +53,26 @@ def json_response
           }
         }
       ]
+    elsif @question_answer && !@next_question_answer
+      topic = @text_component.topic
+      messages =  [
+          {
+          attachment: {
+            payload: {
+              template_type: "button",
+              text: "#{content}",
+              buttons: [
+                {
+                  type: "show_block",
+                  block_name: "cont_" + topic.name,
```

Comment:
I would return a 404 error, if no topic exists. Is it actually allowed for a text_component to not have a topic?